tests/sympc/tensor/mpc_tensor_test.py

```diff
@@ -328,7 +329,10 @@ def test_generate_shares_config(get_clients) -> None:
     assert sum(shares_from_share_tensor) == sum(shares_from_secret)
 
 
-@pytest.mark.parametrize("protocol", ["FSS"])
+fss_obj = Protocol.registered_protocols["FSS"]()
```

Comment:
SHould `fss_obj` be called `fss_protocol`?

Reply:
Yeah that would be apt